Mon Jun 26 22:45:58 CEST 2006  VMiklos <vmiklos@frugalware.org>
  * libfwxwmconfig: it's normal if there is no symlink already there
diff -rN -u old-frugalwareutils/libfwxwmconfig/libfwxwmconfig.c new-frugalwareutils/libfwxwmconfig/libfwxwmconfig.c
--- old-frugalwareutils/libfwxwmconfig/libfwxwmconfig.c	2006-06-26 23:06:05.000000000 +0200
+++ new-frugalwareutils/libfwxwmconfig/libfwxwmconfig.c	2006-06-26 23:06:05.000000000 +0200
@@ -149,8 +149,7 @@
 	snprintf(oldpath, PATH_MAX, "xinitrc.%s", name);
 	snprintf(newpath, PATH_MAX, "%s/xinitrc", _FWXWM_XINITDIR);
 
-	if(unlink(newpath)==-1)
-		return(0);
+	unlink(newpath);
 	if(symlink(oldpath, newpath)==-1)
 		return(0);
 	return(1);

